python/lsst/meas/algorithms/findCosmicRaysConfig.py

```diff
@@ -33,7 +33,7 @@ class FindCosmicRaysConfig(pexConfig.Config):
     nCrPixelMax = pexConfig.Field(
         dtype=int,
         doc="maximum number of contaminated pixels",
-        default=10000,
+        default=1000000,
```

Comment:
Can you explain this change?  I see that this is the obs_subaru value, and that obs_decam also used a higher value, so I can see the underlying logic in relation to this ticket, but this value must be too large.  For LSSTCam, this amounts to 1 CR pixel per 4x4 grid.  HSC and DECam have double the contaminated area.  I can't imagine any image with that level of contamination being scientifically useful.  I'll dig through the obs_subaru git history to find the reason for the high value, but I just want to raise this issue.

Reply:
The large value was also set in obs_lsst, at the root-level `characterizeImage` override, so it's the defacto default now anyway (modulo the small DECam difference). I'd love to have some rationale for why it is what it is, and whether to make it something better. The goal of this ticket is to just make overrides that are defacto defaults into actual defaults.

Reply:
I've added https://jira.lsstcorp.org/browse/DM-40586 as sorting out the correct value is out of scope here.